app/components/Sidebar/components/DocumentLink.js

```diff
@@ -96,7 +99,7 @@ class DocumentLink extends React.Component<Props> {
         this.isActiveDocument())
     );
     const document = documents.get(node.id);
-    const title = node.title || "Untitled";
+    const title = node.title || t("Untitled");
```

Comment:
Nice benefit that we didn't store "Untitled" in the database here!

Reply:
Indeed :)